packages/core/src/Tooltip/Tooltip.js

```diff
@@ -1,5 +1,6 @@
 import propTypes from '@dhis2/prop-types'
 import { colors, layers } from '@dhis2/ui-constants'
+import PropTypes from 'prop-types'
```

Comment:
`import propTypes from '@dhis2/prop-types'`
should be enough

Reply:
I used the react library so I wouldn't have to revisit this once the storybook changes are merged 😅 is it okay to leave it this way?